client/package.json

```diff
@@ -25,7 +25,7 @@
     "react-scripts": "^2.1.3",
     "react-table": "^6.8.6",
     "redux": "^4.0.0",
-    "redux-form": "^7.4.2",
+    "redux-form": "^8.1.0",
```

Comment:
upgrade not necessarily required - I tried upgrading when I couldn't get initialValues to work, but it turned out the Redux Form version was the sisue